### PR TITLE
Feature api validate

### PR DESCRIPTION
**Pull request by askarovit**

### CODE REVIEW

File: api/controller/transaction/index.ts [hunk 1]
```diff
@@ -22,11 +24,15 @@ export class TransactionController {
...
     createTransaction(message: Message, socket: any) {
         // TODO: remove this logic from API
-        message.body.data.trs.createdAt = Date.now() / 1000;
-        console.log('MESSAGE: ', message);
-        SocketMiddleware.emitToCore(message, socket);
+        if (message.isValid) {
+            message.body.data.trs.createdAt = Date.now() / 1000;
+            SocketMiddleware.emitToCore(message, socket);
+        } else {
+           console.log('Not VALID');
```
Comment (Verbalist) [inline api/controller/transaction/index.ts]: this logic must be catch in decorator

File: api/controller/transaction/index.ts [hunk 2]
```diff
@@ -22,11 +24,15 @@ export class TransactionController {
     }
 
     @RPC(CREATE_TRANSACTION)
+    @validate(SCHEMA_CREATE_TRANSACRION)
```
[the remark this answers is not in the document]
Comment (askarovit) [reply api/controller/transaction/index.ts]: I think yes. it works

File: api/controller/transaction/index.ts (hunk 1 above)
Comment (askarovit) [reply api/controller/transaction/index.ts]: Ок, I will fix

File: api/controller/transaction/index.ts [hunk 3]
```diff
@@ -22,10 +24,8 @@ export class TransactionController {
...
```
Comment (axelsheva) [inline api/controller/transaction/index.ts]: `TRANSACRION` -> `TRANSACTION`

**Pull request closed without merging**